src/current/cockroachcloud/egress-perimeter-controls.md

```diff
@@ -10,7 +10,7 @@ cloud: true
 This page describes how Egress Perimeter Controls can enhance the security of {{ site.data.products.dedicated }} clusters, and gives an overview of how to manage a cluster's egress rules.
 
 {{site.data.alerts.callout_info}}
-During [limited access](/docs/{{site.versions["stable"]}}/cockroachdb-feature-availability.html), Egress Perimeter Controls are not supported for {{ site.data.products.dedicated }} clusters on Azure. Refer to [{{ site.data.products.dedicated }} on Azure](cockroachdb-dedicated-on-azure.html).
+Egress Perimeter Controls are not supported for {{ site.data.products.dedicated }} clusters on Azure ([limited access](/docs/{{site.versions["stable"]}}/cockroachdb-feature-availability.html)). Refer to [{{ site.data.products.dedicated }} on Azure](cockroachdb-dedicated-on-azure.html).
```

Comment:
```suggestion
Egress Perimeter Controls are not supported for {{ site.data.products.dedicated }} clusters on Azure ([limited access](/docs/{{ site.current_cloud_version }}/cockroachdb-feature-availability.html)). Refer to [{{ site.data.products.dedicated }} on Azure](cockroachdb-dedicated-on-azure.html).
```

I know you didn't cause this nit, but we should be using `{{ site.current_cloud_version }}` for all links, because there's usually a week (or longer) gap where stable and the current Dedicated version are different.

Reply:
Done